lib/templates/angular.html

```diff
@@ -16,14 +18,6 @@
   	<meta name="apple-mobile-web-app-capable" content="yes">
   	<meta name="apple-mobile-web-app-status-bar-style" content="black">
 
-    <link rel="shortcut icon" href="/assets/img/icons/favicon.png">
-    <link rel="apple-touch-icon-precomposed" sizes="1040x1040" href="/assets/img/icons/apple-touch-icon-1040-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="1024x1024" href="/assets/img/icons/apple-touch-icon-1024-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/assets/img/icons/apple-touch-icon-144-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="/assets/img/icons/apple-touch-icon-114-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/assets/img/icons/apple-touch-icon-72-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" href="/assets/img/icons/apple-touch-icon-57-precomposed.png" />
-
```

Comment:
I am removing them only on the CLI - the paths that they have been set to never worked, and all I remeber is seeing a 404 from time to time on the CLI. So I have added an orange favicon.ico with 'CLI' to 'lib/public/', linked it [here](https://github.com/getbeyond/beyond-cli/blob/bd7eced394b7289dccd91a07fe8b73a6cbfdb787/lib/templates/angular.html#L5) and enabled this folder as static assets [here](https://github.com/getbeyond/beyond-cli/blob/bd7eced394b7289dccd91a07fe8b73a6cbfdb787/lib/runner-process.js#L182)

This way we should now have an orange icon with 'CLI' on the tab when running something with cli - easier to find it if you have 100 tabs open ;-) If you or @mbebas think those favicons did actually work, let me know, but to me it seems they did nothing because these assets were not served by the cli.

Reply:
This makes sense. Thank you Kuba.